bindata/network/ovn-kubernetes/common/002-rbac.yaml

```diff
@@ -13,16 +13,22 @@ metadata:
 rules:
 - apiGroups: [""]
   resources:
-  - endpoints
```

Comment:
How does the unidler work? (I forget). Does the node update the Endpoints object?

Reply:
When someone connects to an idled service, ovn-kubernetes emits an Event which is observed by openshift-controller-manager, which removes the annotation and scales up the service.